### PR TITLE
Added Rain's CCV profile

### PR DESCRIPTION
**Pull request by rainxfan**

### CODE REVIEW

File: about/people.yml [hunk 1]
```diff
@@ -366,3 +366,11 @@ data:
...
     bio: Galen is passionate about building usable, flexible, and accessible web applications. Outside of work, he loves playing music, playing tennis and squash, and hiking. 
     image: galen_main.jpg
+  - name: Rain Fan
+    type: Full Time
+    team: CCV Operations
+    title: Research Computing Consultant
+    github_username: rainxfan
+    brown_directory_uuid: D5bc11b8-b031-4402-8f40-d08ccee18b60
+    bio:
+    image: rain_main.jpg
```
Comment (paulstey) [inline about/people.yml:376]: It looks like the file extension here should be `.jpeg`. Either that, or we should rename the image itself.

**Review by paulstey — CHANGES_REQUESTED** [review]
Hi Rain! Thanks so much for your work on this! It looks like your image isn't showing up because the filename uses the `.jpeg` extension, but the `people.yml` file uses the `.jpg` extension. We should me sure those agree.

**Comment by yangliu2009** [comment]
The two pictures work now. Thanks!

**Review by yangliu2009 — APPROVED** [review]
Uploading the two jpg pictures fixed the issue. Thanks, Paul and Rain.

**Review by paulstey — APPROVED** [review]
This looks great, Rain! Thank you!

**Pull request merged**